짝수와-홀수.js

```diff
@@ -0,0 +1,11 @@
+function solution(num) {
+    if (num % 2 == 0) {
+        return "Even";
+  } 
+    else {
+        return "Odd";
+  }
+}
+
+console.log("결과 : " + solution(2));
+console.log("결과 : " + solution(3));
```

Comment:
마찬가지로 console.log는 지워주시고 return으로 결과값 도출해 주세요

잘 푸셨는데 if else문은 아래와 같이 삼항연산자로도 사용이 가능해요
추가적으로 삼항연산자까지 공부해 보시면 좋을 것 같아요 :)

```suggestion
function solution(num) {
    let answer = "Even";
    (num % 2 == 0) ? answer : answer = "Odd";
    
    return answer;
}
```